src/navigation/initialRoute.ts

```diff
@@ -0,0 +1,4 @@
+import { createContext } from 'react';
+import Routes from './routesNames';
+
+export const InitialRouteContext = createContext<string>(Routes.WELCOME_SCREEN);
```

Comment:
curious about the change from `null` as the default value to the `WELCOME_SCREEN` here if it's needed or if it'll work the same with `null` as it was and just passed in Context type `<string | null>`

Reply:
Didn't see any change in behavior personally